### ISSUE TITLE
Update ICSPR IP address range/access

### ISSUE THREAD

**Issue by sbarbosadataverse** [IQSS/dataverse#1920]
ICPSR Direct Permitted Addresses

Institution Name: Harvard University

Institution Number: 45

Last Changed: Unrecorded

Allowed IP Addresses:

128.103.0.0/16
131.142.0.0/16
134.174.14.0/23
134.174.140.0/22
134.174.144.0/20
134.174.16.0/23
134.174.160.0/21
134.174.168.0/22
134.174.172.0/23
134.174.174.0/24
134.174.178.0/23
134.174.180.0/22
134.174.184.0/21
134.174.193.0/24
134.174.4.0/22
140.247.0.0/16
199.94.0.0/19
199.94.32.0/20
207.86.182.0/24
212.171.47.146/32

CIDR

If the above IP ranges look odd to you, it's because they're expressed using CIDR, a format rapidly being adopted by IT organizations and universities.

To convert CIDR to traditional IP ranges (or vice versa), please use this CIDR-IP Converter.

**Comment by kcondon** [comment]
@sbarbosadataverse This is an old ticket and I'm wondering what it means and whether it is still relevant. Can you comment? I'd like to close it or work on it. Thanks!

**Comment by sbarbosadataverse** [comment]
@scolapasta @mcrosas 
Gustavo, I don't think we can just close this.
This has to do with harvesting ICPSR data/or how Harvard accesses their data.
Merce may know more, but I captures what was put into a ticket and I don't think this is resolved.

**Comment by kcondon** [comment]
I think this is the Harvard IP group addresses that is allowed file access to harvested ICPSR harvested datasets. We do not currently have harvesting or officially support IP groups to my knowledge so not sure what to do with this information, it's not a bug but really something to consider when we restore harvesting and ICPSR in particular.

________________________________

**Comment by sbarbosadataverse** [comment]
That's fine, but why assign it to me?
I can't do this when we do have harvesting in place. I think it should go
back to whomever is dealing with harvesting, and we can change it from bug
to whatever is more appropriate

On Fri, Jan 29, 2016 at 11:03 AM, kcondon <notifications@github.com> wrote:

>
> I think this is the Harvard IP group addresses that is allowed file access
> to harvested ICPSR harvested datasets. We do not currently have harvesting
> or officially support IP groups to my knowledge so not sure what to do with
> this information, it's not a bug but really something to consider when we
> restore harvesting and ICPSR in particular.
>
> ________________________________
> From: sbarbosadataverse [notifications@github.com]
> Sent: Friday, January 29, 2016 10:16 AM
> To: IQSS/dataverse
> Cc: Condon, Kevin M
> Subject: Re: [dataverse] Update ICSPR IP address range/access (#1920)
>
>
> @scolapasta<
> https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_scolapasta&d=CwMCaQ&c=WO-RGvefibhHBZq3fL85hQ&r=TUpjWt9sVfaAC8ETCY_cDPtqJKl7s242PLg6-Wx6UpM&m=S8fKwRxtXC-azoT71FT1Yt1naeMjlyNmIvx8JItvmvw&s=qSHXJe1yQ5iL9_y5pIsm6h3EMhbURCxfPlkthxGHAp8&e=>
> @mcrosas<
> https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_mcrosas&d=CwMCaQ&c=WO-RGvefibhHBZq3fL85hQ&r=TUpjWt9sVfaAC8ETCY_cDPtqJKl7s242PLg6-Wx6UpM&m=S8fKwRxtXC-azoT71FT1Yt1naeMjlyNmIvx8JItvmvw&s=skca1xiKIu665O5UScjGAwSLTItDrH3-BP0WMAbGQQM&e=
> >
> Gustavo, I don't think we can just close this.
> This has to do with harvesting ICPSR data/or how Harvard accesses their
> data.
> Merce may know more, but I captures what was put into a ticket and I don't
> think this is resolved.
>
> —
> Reply to this email directly or view it on GitHub<
> https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_IQSS_dataverse_issues_1920-23issuecomment-2D176803982&d=CwMCaQ&c=WO-RGvefibhHBZq3fL85hQ&r=TUpjWt9sVfaAC8ETCY_cDPtqJKl7s242PLg6-Wx6UpM&m=S8fKwRxtXC-azoT71FT1Yt1naeMjlyNmIvx8JItvmvw&s=ZeJyNWg-cf4Hp6glM3BA_l0KxjGYmSBX2g3tvWeoMtk&e=
> >.
>
> —
> Reply to this email directly or view it on GitHub
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_IQSS_dataverse_issues_1920-23issuecomment-2D176837812&d=CwMFaQ&c=WO-RGvefibhHBZq3fL85hQ&r=8R6PzVqt1PEocauQgZMGXsGz29-nb19M7eqlo1d8EVs&m=im6NMshvIkEqS6O-5rqJ4Du7m_rpeAJ6QRbxNpnoWjI&s=Z7zILxL-8CPfpChlnqmsdooRxRyTgk3ZWQv_xyu5vFI&e=>
> .
>



-- 
Sonia Barbosa
Manager of Data Curation, IQSS Dataverse Network
Manager of the Murray Research Archive, IQSS
Data Science
Harvard University

Dataverse 4.0 is now available for use!
http://dataverse.harvard.edu

All test dataverses should be created  in 4.0 Demo!
http://dataverse-demo.iq.harvard.edu/

Join our Dataverse Community!
https://groups.google.com/forum/#!forum/dataverse-community

**Comment by kcondon** [comment]
Thanks for your help Sonia, and your time in looking at this. It's much appreciated ;)
I sent it to you to look at because I wasn't sure it was still an issue and wanted you to comment. Now that I think I know what the issue is, we can remove your assignment. @scolapasta under the new ticketing workflow, should this just be unassigned for now and perhaps linked to the harvesting ticket(s)?

________________________________

**Comment by scolapasta** [comment]
I'm going to go ahead and close this.

While we don't yet harvesting via OAI, it was possible to manually harvest ICPSR via the APIs, and we did. And more importantly, we changed the the way we interact with harvested datasets: In 3.6, we needed these IP ranges because DVN confirmed you had permission and then downloaded the file from ICPSR (or other harvest source) for the user. In 4.0, we don't do that anymore, we sent you directly to the harvest source, and permission access is administered by the harvest source itself.
